src/zepto.js

```diff
@@ -51,9 +51,9 @@ var Zepto = (function() {
 
   zepto.matches = function(element, selector) {
     if (!selector || !element || element.nodeType !== 1) return false
-    var matchesSelector = element.webkitMatchesSelector || element.mozMatchesSelector ||
+    var matches = element.matches || element.webkitMatchesSelector || element.mozMatchesSelector ||
```

Comment:
I'm sorry for making a mistake.
Last night , I downloaded Opera12 and got a test,  oMatchesSelector is defined in Opera12. In Opera15 , webkitMatchesSelector is defined as a native method, but no oMatchesSelector.
If we need to support Opera12, I just keep it here. 
I'm sorry for wasting your time.